### ISSUE TITLE
When launching from Clin, please show the GTR search page by default (rather than the landing page)

### ISSUE THREAD

**Issue by tonydisera** [iobio/genepanel.iobio#229]

**Comment by adityaekawade** [comment]
@tonydisera  @AlistairNWard , 
Can you please review and tell if you see GTR as default when launched from clin?

**Comment by tonydisera** [comment]
Thank you!  I will give it a try.

**Comment by tonydisera** [comment]
I pulled down Dev, and removed node_modules.

npm install gives this error:

pm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@4.5.3 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@4.5.3 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.


Then npm start gives a bunch of errors about node-sass:

ERROR in ./~/css-loader?sourceMap!./~/vue-loader/lib/style-compiler?{"vue":true,"id":"data-v-7fa3c4d8","scoped":false,"hasInlineConfig":false}!./~/sass-loader/lib/loader.js?indentedSyntax!./~/vue-loader/lib/selector.js?type=styles&index=1!./client/app/components/pages/Phenolyzer.vue
Module build failed: Error: Cannot find module 'node-sass'

**Comment by tonydisera** [comment]
Yep, that worked.  I wonder why package.json devDependencies didn't cause it to install.  Can you find out?  We want to make sure this doesn't hit outside developers that download our source and try to build it.  Thanks!

When launched from clin, genepanel did go to GTR.  Would you look into styling the title bar using a white background when launched from clin, similiar to gene.iobio?  

<img width="1433" alt="screen shot 2018-08-18 at 9 13 49 am" src="https://user-images.githubusercontent.com/8420103/44300611-2fdcef00-a2c7-11e8-9f38-1266f2f874e5.png">

**Comment by adityaekawade** [comment]
Hi @tonydisera . I looked into the issue and this is much more intriguing! 

There are two possible reasons why this issue was caused: 
1. The difference in the version number of `npm` or `node-sass` module in our machines.  I looked into the version number of node-sass module of gene and clin on GitHub repos and it was different from the version number of panel app. 
2. Node-sass module can also cause issues while installing (https://stackoverflow.com/a/46515489/2150900)

Deleting package.lock.json before `npm install` would likely install everything properly.

**Comment by tonydisera** [comment]
Excellent!  Thank you Adit!

**Comment by AlistairNWard** [comment]
When you go to candidate genes in clin, the landing page opens, and then switches to GTR. Can we go directly to GTR? I know @tonydisera fixed this in gene, so the landing page is skipped when opening gene from mosaic.
